### PR TITLE
Update kubespray.md

### PR DESCRIPTION
**Pull request by wangxf1987**
upgrade that deploy model desciption

### CODE REVIEW

File: content/zh/docs/setup/production-environment/tools/kubespray.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ Kubespray 能够自定义部署的许多方面：
   * {{< glossary_tooltip term_id="cri-o" >}}
 * Certificate generation methods
 -->
-* 选择部署模式： kubeadm 或非 kubeadm
+* 选择部署模式： kubeadm 或非 kubeadm （该方式在v2.8版本以后不再支持）
```
Comment (chenxuc) [inline content/zh/docs/setup/production-environment/tools/kubespray.md:147]: @wangxf1987 Thanks for your first PR here!
I don't see the statement for "v2.8版本以后不再支持" in the English version of the page https://kubernetes.io/docs/setup/production-environment/tools/kubespray/ .

Could you please provide the reference document for this change? If this change is needed, we should update the English version first, then other languages will pick up the change from the English page.

Comment (wangxf1987) [reply content/zh/docs/setup/production-environment/tools/kubespray.md:147]: push a PR on the en doc, please review it. thx

**Comment by wangxf1987** [comment]
The non-kubeadm model deprecated non-kubeadm deployment starting from v2.8, please refer to https://github.com/kubernetes-sigs/kubespray/blob/master/docs/comparisons.md

**Comment by chenrui333** [comment]
/hold

relates to #28831

**Comment by tengqm** [comment]
/close
Closing. Feel free when the upstream fix is accepted.

**Pull request closed without merging**